lib/spoom/cli/commands/coverage.rb

```diff
@@ -75,9 +77,11 @@ def timeline
               Bundler.with_clean_env do
                 next unless bundle_install(path, sha)
                 snapshot = Spoom::Coverage.snapshot(path: path)
+                snapshot.sorbet_version = Spoom::Sorbet.srb_version(path: path) if snapshot
               end
             else
               snapshot = Spoom::Coverage.snapshot(path: path)
+              snapshot.sorbet_version = Spoom::Sorbet.srb_version_from_gemfile_lock(path: path)
```

Comment:
In this case, maybe we can optionally flag that the version should be parsed from the gemfile:
```suggestion
              snapshot = Spoom::Coverage.snapshot(path: path, version_source: :gemfile)
```
kind of thing.